src/GslibIO.jl

```diff
@@ -155,4 +199,58 @@ function save(file::File{format"GSLIB"}, sdata::AbstractData)
        propnames=vars)
 end
 
+# low level function for saving `data` to a legacy GSLIB format using `varnames` as variable names
+function save_legacy(filename::AbstractString, data::AbstractMatrix, varnames::NTuple; 
+                     na=-999.0, header="This file was generated with GslibIO.jl")
+  nvars = size(data, 2)
+  @assert nvars == length(varnames) "The length of variable names must be equal to the number of variables"
+
+  open(filename, "w") do f
+    write(f, "$header\n")
+
+    write(f, "$nvars\n")
+    for v in varnames
+      write(f, "$v\n")
+    end
+
+    # handle missing values
+    replace!(data, NaN=>na)
+
+    writedlm(f, data, ' ')
+  end
+end
+
+"""
+    save_legacy(file, sdata)
+
+Save spatial data `sdata` to `filename` using standard GSLIB format. It replaces NaNs with `na` values
+and it uses `coordnames` as the given variable name to coordinates
+"""
+function save_legacy(filename::AbstractString, sdata::SpatialData; coordnames=(:x, :y, :z), na=-999.0)
```

Comment:
```suggestion
function save_legacy(filename::AbstractString, sdata::AbstractData; na=-999.0)
```
We want to be able to save any spatial data, hence `AbstractData`. Also, we don't need to specify the names of the coordinates because this information is available in the object already 👍🏽

Reply:
I cannot find how to get the coordinate names from the object. I think the `coordnames` to write into the file it should be a user decision.

Reply:
You are correct, I think we deleted the features related to the coordinate names in a refactoring of the codebase. Let's keep the option then, thanks for double checking.